src/Front-end/Messages.js

```diff
@@ -1,70 +1,204 @@
 import "./Messages.css"
+import React, { useState, useRef, useEffect } from "react";
+import ReactDOM from 'react-dom'
 import NavBar from "./NavBar";
-
 import { pageTheme } from "./util";
+import { Typography, Box, TextareaAutosize ,Container, Grid, Paper, TextField, ThemeProvider, List, ListItem, Checkbox, ListItemText, Slider, Input, MenuItem, Select, Stack, Button } from '@mui/material';
+import axios from 'axios';
+import { auth } from "../login/loginconfig";
+
+
+// }
+// const handleNewChat = () =>{
+//     axios.newChat('http://localhost:8000/api/messages',data).then(res => {
+//         // clear the form fields
+//         // notify user that their listing was posted
+//       }).catch(error => {
+//         console.error('Error posting data:', error);
+// })}
+
+//sample messages only sent by the current user
+
+//param to pass in through axios, need component to operate with verified email address
+export const USERNAME = "bruh1"
+
+export const USERLIST = ["Buyer1", "Seller1", "Buyer2", "Selling2"]
+
+const messages = [
```

Comment:
Can this dummy data be deleted to clean things up or is it still in use?

Reply:
It can be removed